### PR TITLE
Add switchover and switchover-rollback sections to paperclip migration

### PR DESCRIPTION
**Pull request by jsugarman**
#### What
Add switchover and switch over rollback sections to paperclip migration

#### Why

To document the process of switching over from paperclip
to active storage and the mechanism to revert/rollback
from active storage to paperclip.

### CODE REVIEW

File: docs/active_storage_migration.md [hunk 1]
```diff
@@ -324,6 +324,56 @@ Storage migration is complete this last ticket needs to be reviewed to see if
 the issue has been resolved. This can be done by checking for redraft failures
 due to timeouts in Kibana logs, for example.
 
+## Switchover
+
+> **_NOTE:_** This switchover from paperclip to active storage assumes that checksums have been calculated for the model being activated and the target environment has latest master on it - with the `storage.rake` tasks is available on it.
```
Comment (jrmhaig) [inline docs/active_storage_migration.md:329]:
```suggestion
> **_NOTE:_** This switchover from paperclip to active storage assumes that checksums have been calculated for the model being activated and the target environment has latest master on it - with the `storage.rake` tasks available on it.
```

**Review by jrmhaig — APPROVED** [review]
One minor error but otherwise OK.

File: docs/active_storage_migration.md (hunk 1 above)
Comment (jsugarman) [reply docs/active_storage_migration.md:329]: it needs to be plural "with the `storage.rake` task**s** available on it"

Comment (jrmhaig) [reply docs/active_storage_migration.md:329]: I think there is an extra 'is'.

Comment (jsugarman) [reply docs/active_storage_migration.md:329]: oh yeah, friday brain

**Pull request merged**